packages/browser/package.json

```diff
@@ -23,6 +23,7 @@
     "access": "public"
   },
   "dependencies": {
+    "@sentry-internal/feedback": "7.81.0",
```

Comment:
Hmm the next/remix tests are failing -- should I put this at the published alpha package for now?